Added plugin descriptions and links to README.md

cyrusadkisson opened the pull request:
I was unsure about a few of the plugins. Could you add a little information for the ones labeled "need input from brummer"?

cyrusadkisson committed:
Update README.md

Added links and background information for all the different submodules so that users know what's what. Reorganized the page to put background/summary information at the top, build/installation information in the middle and all the pedal images at the bottom.

cyrusadkisson committed:
Merge pull request #1 from cyrusadkisson/cyrusadkisson-patch-1

Update README.md

cyrusadkisson committed:
Update README.md

Added link to Ampeg SVT

cyrusadkisson committed:
Update README.md

brummer10 commented:
Hi

Many thanks for your pull request, it's much appreciated.

To your questions, here we go:

GxTubeDistortion.lv2 -> this one comes from here: http://www.montagar.com/~patj/tubedist.gif

GxVoodoFuzz.lv2  -> simulation impressed by the (*) Voodoo Lab  SuperFuzz pedal. It's basically a Bosstone circuite, followed by the tone control of the (*) FoxToneMachine in parralel with a DarkBooster, finaly ended in a Volume control.

GxVBassPreAmp.lv2 -> Simulation of the 1984 (*) Vox Venue Bass 100 Pre Amp Section.

GxVmk2.lv2 -> (*) Vox mkii preamp ' late-60'ies solid state amp

I think it would be good if we could keep the (*) for the disclaimer and add it in any place were trademarks or companies been mentioned.

cyrusadkisson committed:
Additional submodule info based on brummer's feedback

Also moved the debian module note to the build section.

cyrusadkisson committed:
Removed broken image from Ross dist

cyrusadkisson committed:
Added symbol for "includes master bypass"

Master bypass is an important feature for DAW automation, so I thought it would be helpful to include an at-a-glance indicator here.
